dotcom-rendering/scripts/gen-stories/gen-stories.js

```diff
@@ -0,0 +1,171 @@
+/*
+
+Use: node dotcom-rendering/scripts/gen-stories/gen-stories.js
+
+This script was created as a replacement for storiesOf to generate all of the possible variants
+of our Card and Layout components.
+
+It should be run whenever a new Display, Design, or Theme
+
+*/
+
+const {
+	ArticleDesign,
+	ArticleDisplay,
+	ArticlePillar,
+	ArticleSpecial,
+} = require('@guardian/libs');
+const { writeFile, rm, readFile } = require('fs').promises;
+const path = require('path');
+const { log, success } = require('../env/log');
+
+const STORIES_PATH = path.resolve(__dirname, '..', '..', 'stories');
+const LAYOUT_STORIES_FILE = path.resolve(STORIES_PATH, 'Layout.stories.tsx');
+const CARD_STORIES_FILE = path.resolve(STORIES_PATH, 'Card.stories.tsx');
+
+const CARD_TEMPLATE_HEADER = `
+// File generated using dotcom-rendering/scripts/gen-stories/gen-stories.js
```

Comment:
Worth leaving a note not to edit? [e.g.](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/fixtures/generated/images.ts#L1-L12)